larq_compute_engine/mlir/BUILD

```diff
@@ -16,6 +16,7 @@ gentbl(
     ],
     tblgen = "@llvm-project//mlir:mlir-tblgen",
     td_file = "ir/lce_ops.td",
+    td_includes = ["external/org_tensorflow"],
```

Comment:
I spent hours debugging why the `gentbl` rules weren't working (they complained about not being able to find the TF tablegen files). It turns out that the implementation of `gentbl` was [recently re-written](https://github.com/tensorflow/tensorflow/commit/76e6fd8ad9f4fd683c830118b941c42f90e1ad04#diff-1a9f4c347a8d3612bfaa2b92a030402ffb15a6fe9e7bf78fd0f13f006824cfed) and in the previous version there was a blanket hard-coded include of the TF workspace, which no longer survives. This ugliness is the only way I could work out how to get these working again; the `td_srcs` argument is only used for building the action graph and doesn't get fed into the `tblgen` binary, and the `deps` argument doesn't work either if a filegroup is passed in.

They've also implemented a new Bazel rule [`td-library`](https://github.com/tensorflow/tensorflow/commit/a275a1467fbce174e5bbd3b941733154186c563e#diff-1a9f4c347a8d3612bfaa2b92a030402ffb15a6fe9e7bf78fd0f13f006824cfed) but it's not actually used yet; when it is, I *think* that it will solve our issues as e.g. `tensorflow_ops_td_files` will be implemented as a `td-library` rather than a filegroup, which can then be passed in to `deps`.

Reply:
Thanks for figuring this out! 🚀 
I briefly looked into upgrading as well but didn't make it past this road block.